[CI] Networking job is currently failing

The [functional-networking job is currently failing](https://github.com/gophercloud/gophercloud/actions/workflows/functional-networking.yaml?query=workflow%3Afunctional-networking+event%3Aschedule++) consistently, since Feb 7th (for Yoga) and Feb 22th (for master). We have recently deleted the Yoga job as this release went unmaintained upstream, so let's only focus on the master job.

It is failing with:
```
=== RUN   TestPortsbindingCRUD
    networking.go:35: Attempting to create network: TESTACC-p2isBNwi
    networking.go:42: Successfully created network.
    networking.go:305: Attempting to create subnet: TESTACC-jhqfg6Wo
    networking.go:312: Successfully created subnet.
    portsbinding.go:27: Attempting to create port: TESTACC-8j3XRIFJ
    convenience.go:36: Failure in portsbinding_test.go, line 40: unexpected error "Request forbidden: [POST http://10.1.0.97:9696/networking/v2.0/ports], error message: {\"NeutronError\": {\"type\": \"PolicyNotAuthorized\", \"message\": \"(((rule:create_port and (rule:create_port:fixed_ips and (rule:create_port:fixed_ips:subnet_id))) and rule:create_port:binding:host_id) and rule:create_port:binding:profile) is disallowed by policy\", \"detail\": \"\"}}"
    networking.go:553: Attempting to delete subnet: 4286d14d-4f29-4209-9e6f-72dde8481e3b
    networking.go:560: Deleted subnet: 4286d14d-4f29-4209-9e6f-72dde8481e3b
    networking.go:525: Attempting to delete network: b2e7f78e-bb40-466a-8716-2bf785c3fe7b
    networking.go:532: Deleted network: b2e7f78e-bb40-466a-8716-2bf785c3fe7b
--- FAIL: TestPortsbindingCRUD (0.68s)
```

--- a/.github/workflows/functional-fwaas_v2.yaml
+++ b/.github/workflows/functional-fwaas_v2.yaml
@@ -39,6 +39,10 @@ jobs:
             Q_ML2_PLUGIN_TYPE_DRIVERS=flat,gre,vlan,vxlan
             Q_ML2_TENANT_NETWORK_TYPE=vxlan
             Q_TUNNEL_TYPES=vxlan,gre
+
+            [[post-config|/etc/neutron/policy.d/port_binding.yaml]]
+            "create_port:binding:profile": "rule:admin_only or rule:service_api"
+            "update_port:binding:profile": "rule:admin_only or rule:service_api"
           enabled_services: 'q-svc,q-agt,q-dhcp,q-l3,q-meta,q-fwaas-v2,-cinder,-horizon,-tempest,-swift,-c-sch,-c-api,-c-vol,-c-bak,-ovn,-ovn-controller,-ovn-northd,-q-ovn-metadata-agent'
       - name: Checkout go
         uses: actions/setup-go@v5

